usaspending_api/etl/elasticsearch_loader_helpers/transform_data.py

```diff
@@ -38,18 +38,24 @@ def transform_covid19_faba_data(worker: TaskSpec, records: List[dict]) -> List[d
         award_type = record.pop("award_type")
         generated_unique_award_id = record.pop("generated_unique_award_id")
         total_loan_value = record.pop("total_loan_value")
-        temp_key = f"{disinct_award_key}|{award_id}|{award_type}|{generated_unique_award_id}|{total_loan_value}"
+        obligated_sum = record.get("transaction_obligated_amount") or 0
+        outlay_sum = record.get("gross_outlay_amount_by_award_cpe") or 0
+        temp_key = disinct_award_key
         if temp_key not in results:
             results[temp_key] = {
                 "financial_account_distinct_award_key": disinct_award_key,
                 "award_id": award_id,
-                "award_type": award_type,
+                "type": award_type,
                 "generated_unique_award_id": generated_unique_award_id,
                 "total_loan_value": total_loan_value,
                 "financial_accounts_by_award": list(),
+                "obligated_sum": obligated_sum,
```

Comment:
I recommend following the pattern for `financial_accounts_by_award`. For these two fields simply set the initial value to 0 in the dict, then add the new values below. Doesn't require the else